package.json

```diff
@@ -13,14 +13,10 @@
         "dev": "tsup src/index.ts --format esm,cjs --watch --dts",
         "prepublishOnly": "npm run build"
     },
-    "dependencies": {
-        "js-cookie": "^2.2.1",
-        "lodash": "^4.17.21"
-    },
     "devDependencies": {
-        "@types/js-cookie": "^3.0.2",
-        "@types/lodash": "^4.14.183",
+        "@types/js-cookie": "^2.2.1",
         "@types/node": "^18.7.6",
+        "js-cookie": "^2.2.1",
```

Comment:
I might be a little confused here. Isn't it actually better to **not** bundle deps, but rather let the user's build system take care of it? Just thinking that `js-cookie` could potentially be included twice if the user has another package that also uses `js-cookie`, because it wouldn't be de-duplicated if it's bundled directly, would it? In other words, if we specify `js-cookie` as a dep, and another package specifies `js-cookie` as a dep, then a user's build system would know to resolve to one copy, whereas it wouldn't be able to do that if `js-cookie` is bundled directly?

Reply:
yes, I totally forgot about that. I'll move it back to the `dependencies` property.

Reply:
updated ✅